modules/serverless-install-web-console.adoc

```diff
@@ -15,17 +15,13 @@ This procedure describes how to install and subscribe to the {ServerlessOperator
 image::serverless-search.png[{ServerlessOperatorName} in the {product-title} web console]
 
 . Review the information about the Operator and click *Install*.
+. On the *Install Operator* page:
 +
-image::serverless-operator.png[{ServerlessOperatorName} information]
-
-. On the *Create Operator Subscription* page:
-+
-image::serverless-create-sub.png[Create Operator Subscription page]
+image::serverless-create-sub.png[Install Operator page]
 
 .. The *Installation Mode* is *All namespaces on the cluster (default)*. This mode installs the Operator in the default `openshift-operators` namespace to watch and be made available to all namespaces in the cluster.
 .. The *Installed Namespace* will be `openshift-operators`.
-.. Select the *4.4* channel as the *Update Channel*. The *4.4* channel will enable installation of the latest stable release of the {ServerlessOperatorName}.
-// ... The *preview-4.4* channel enables installation of the latest preview version of the {ServerlessOperatorName}, which may contain features that are not yet available from the *4.4* update channel.
+.. Select the *4.5* channel as the *Update Channel*. The *4.5* channel will enable installation of the latest stable release of the {ServerlessOperatorName}.
```

Comment:
TBH, I don't know if this will be true once we publish an operator for OCP 4.6. The OCP should automatically offer only operators that are targeted for OCP 4.6 so there might not be any choices. But we can probably integrate this. It's better than what we had before. And later change again if needed.

Reply:
I've just checked what we will have for 1.9.0 in nightly OCP 4.6 and it's showing only the "4.5" channel. Nothing else. I believe later it will be changed to 4.6. 
When installing Serverless 1.8.0 there are multiple channel options.
I'd say the text you have right now is good. And later might be changed to just "4.6"